deltachat-repl/README.md

```diff
@@ -0,0 +1,60 @@
+# Delta Chat REPL
+
+This is a [REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop) frontend built on top of Delta Chat core.
+Its purpose is to help with testing during development, it is not meant for end users.
+
+Dependencies:
+- if you want to use `getqr` you need `qrencode` (To install use your systems package manager)
```

Comment:
```suggestion
- if you want to use `getqr`, you need `qrencode` (To install, use your system's package manager)
```
Though i'm not good at punctuation, so don't trust me too much

Reply:
```suggestion
- If you want to use `getqr` you need `qrencode` (To install, use your system's package manager)
```
language tool suggests this.